schema-dex-amm-extended.graphql

```diff
@@ -822,39 +1141,42 @@ type Deposit @entity {
   account: Account!
 
   " The user position changed by this event "
-  position: Position!
+  position: Position
 
   " The pool involving this event "
   pool: LiquidityPool!
 
+  # lower tick of position
+  tickLower: BigInt
+
+  # upper tick of position
+  tickUpper: BigInt
+
   " Block number of this event "
   blockNumber: BigInt!
 
   " Timestamp of this event "
   timestamp: BigInt!
 
+  " Amount of liquidity minted "
+  liquidity: BigInt!
+
   " Input tokens of the pool. E.g. WETH and USDC to a WETH-USDC pool "
   inputTokens: [Token!]!
 
-  " Output token of the pool. E.g. the UNI-LP token "
-  outputToken: Token
-
   " Amount of input tokens in the token's native unit "
   inputTokenAmounts: [BigInt!]!
 
-  " Amount of output tokens in the token's native unit "
-  outputTokenAmount: BigInt
-
   " USD-normalized value of the transaction of the underlying (e.g. sum of tokens deposited into a pool) "
   amountUSD: BigDecimal!
 }
 
-type Withdraw @entity {
+type Withdraw @entity(immutable: true) {
   " withdraw-{ Transaction hash }-{ Log index }"
```

Comment:
Why prepend withdraw and deposit?

Reply:
Agreed that it is not necessary. Updated!